packages/mapviewer/src/utils/composables/useTippyTooltip.js

```diff
@@ -110,15 +119,21 @@ export function useTippyTooltip(
     }
 
     function refreshTippyAttachment() {
+        // if the selector is a function, call it.
```

Comment:
This was a byproduct of debugging the tooltips. If we don't have this we sometimes get too many tooltip instances in the DOM because the selector was too "wide".